doomworld_downloader/doomworld_downloader.py

```diff
@@ -48,12 +52,19 @@ class Post:
     attachments: dict
     links: dict
     post_text: str
+    parent: Thread
 
 
 def get_link_elems(link_elems, extract_link=False):
```

Comment:
Should this be `get_links` instead, since you give it `link_elems` and get back the links? :sweat_smile: